website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+
+## Implementation Guidance
+
+This section provides sample JSON-LD snippets for each DIA component.
+
+### Digital Identity Anchor
+
+The Digital Identity Anchors is a Verifiable credential. Please refer to [DPP VC Guidance](DigitalProductPassport.md#verifiable-credential) for information about the use of the verifiable credentials data model for UNTP.  THe issuing party for the VC MUST be the authority that owns or operates the identity register. 
+
+### Registered Identity
+
+The registered identity class represents the registry member. For example, in a national business register, the registered identity would be one of the registered businesses.
+
+* `type` MUST contain constant value array `["RegisteredIdentity", "Identifier"]` which declares to linked data processors that this is a registered identity which is a sub-type of identity.
+* the `id` MUST contain the globally unique member identifier as maintained in the register.  Often this will be a web link to the member entry.
+* `name` is the human readable registered entity name and SHOULD match the registered name as it appears in the authoritative register.
+* `registeredId` contains the simple identifier value that is unique within the register (but may not be globally unique) for example the VAT registration number.
+* `idScheme` identifies the authoritative register.  If the identity scheme is registered with UN/CEFACT then the `idScheme.id` MUST match the `identityRegister.id` in the UN/CEFACT scheme register.
+* `registerType` is a coded value that allows verifiers to distinguish between different DIA [use cases](#use-cases)
+* `verifiedDIDList` contains an array of DIDs that the member has requested (and the registrar has verified) to be linked to the registeredId of the member.
+* `registrationScopeList` contains a list of URIs that define the scope of the member registration. The values are very specific to the register. For example a national business register would typically have a controlled vocabulary of entity types (eg [Australian Entity Types](https://abr.business.gov.au/Help/EntityTypeList))
+
+
+```json
+  "credentialSubject": {
+    "type": [
+      "RegisteredIdentity",
+      "Identifier"
+    ],
+    "id": "https://sample-register.gov/90664869327",
+    "name": "Sample business Ltd",
+    "registeredId": "90664869327",
+    "idScheme": {
+      "type": [
+        "IdentifierScheme"
+      ],
+      "id": "https://sample-register.gov",
+      "name": "Sample National Business Register"
+    },
+    "registerType": "Business",
+    "verifiedDIDList": [
+      "did:web:samplecompany.com/123456789",
+      "did:ethr:0x5:0xf3beac30c498d9e26865f34fcaa57dbb935b0d74"
+    ],
+    "registrationScopeList": [
+      "https://sample-register.gov/EntityType?Id=00019"
+    ]
+  }
+```
+
+## DIA Discovery
+
+DIA credentials SHOULD be discoverable from either identifier:
+
+* Given a DID (eg as the issuer of a DPP) via DID document `service` end point.
```

Comment:
Do we return the credential or a presentation? I think a presentation would make sense, and I would refer to the linked-vp spec since this is exactly what it looks to do. TDW will likely be merged with did:web in the near future, and there's the [implicit `whois` service](https://identity.foundation/trustdidweb/#whois-linkedvp-service), which can also be listed explicitly. This gives a clear instruction as to what this service should resolve to.